airflow/providers/google/cloud/hooks/compute.py

```diff
@@ -405,10 +405,11 @@ def get_instance_address(
         instance_info = self.get_instance_info(project_id=project_id, resource_id=resource_id, zone=zone)
         if use_internal_ip:
             return instance_info["networkInterfaces"][0].get("networkIP")
-
-        access_config = instance_info.get("networkInterfaces")[0].get("accessConfigs")
-        if access_config:
-            return access_config[0].get("natIP")
+        network_interfaces = instance_info.get("networkInterfaces")
+        if network_interfaces:
+            access_config = network_interfaces[0].get("accessConfigs")
+            if access_config:
+                return access_config[0].get("natIP")
```

Comment:
Shame https://www.python.org/dev/peps/pep-0505/ doesn't exist yet.

Reply:
It’s not likely to ever exist

Reply:
Yeah fair.

Reply:
Why? I really would like to have some of those concepts in PEP 505 implemented. Are there good reasons it's not?